src/_pytest/config/__init__.py

```diff
@@ -29,6 +28,11 @@
 from _pytest.outcomes import Skipped
 from _pytest.warning_types import PytestConfigWarning
 
+if sys.version_info >= (3, 8):
```

Comment:
Could you please move this to `_pytest/compat.py` and import `metadata` from there?

Reply:
sure

Reply:
the import also seems a little weird, I'd probably (if it were just python3.8` use `import importlib.metadata`, so maybe let's do:

```python
if sys.version_info >= (3, 8):
    import importlib.metadata as importlib_metadata
else:
    import importlib_metadata
```

`metadata` seems too-common of a variable name to import as a module, might collide at some point

Reply:
no problem. especially since it's going to be in the compat module.